client/src/util/helpers.ts

```diff
@@ -168,6 +169,14 @@ export default {
     return max;
   },
 
+  getMostCommonElementsInArray(array: string[], numberOfResults: number): string[] {
+    const elementsByFrequency = _.countBy(array);
+    const sortedElementsByFrequency = Object.entries(elementsByFrequency).sort(
+      (a, b) => b[1] - a[1]
+    );
+    return sortedElementsByFrequency.slice(0, numberOfResults).map((a) => a[0]);
```

Comment:
Can you explain to me what the map() is doing here?

Reply:
Yes, so the `sortedElementsByFrequency` const is an array of string + number pairs with the string being the element value (like landlord name) and the number being the frequency. Since the overall helper function just wants to return a simple array, this last map is essentially deleting the record of the frequency and just keeping the element values, in order. Does that make sense?

Reply:
Gonna add a comment.